pipeline/lib/pipeline-stack.ts

```diff
@@ -54,7 +56,7 @@ export class PipelineStack extends cdk.Stack {
       input: sourceArtifacts,
       environmentVariables: {
         S3_BUCKET: {value: apiArtifactBucket.bucketName},
-        GIT_BRANCH: {value: sourceAction.variables.branchName}
+        GIT_BRANCH: {value: gitBranch}
```

Comment:
Why can't we keep the reference to the previous reference?

Reply:
I tried to do it with the codestar object, but I don't think it is supported.
Do you know if there is any other way to re-use that info?